Implement Parameter

dalderliesten opened the pull request:
Added the fire outbreak location parameter to the mobile client.  Must be tested and merged in conjunction with the server/Oculus client PR.

Commit:
Implement Parameter

Added the fire outbreak location parameter to the mobile client.

dmalarev commented on app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/minigame/FireOutbreakActivity.java:
```diff
@@ -168,5 +179,4 @@ public void onPlasmaPress(View view) {
             validateAction(-3);
```
wouldn't it be better to just use one number for the else cases, instead of -1,-2,-3?

jessetilro replied:
Why send invalid numbers at all? You shouldn't have to send invalid numbers to the server, only send messages when they are potentially valid solutions. If something has to be in order locally with the minigame before it can be submitted, just verify it locally, don't bother the server with a message that is always invalid. It simply increases network load. The purpose of the server is to validate messages that are potentially right. Does this story make sense?

dalderliesten replied:
Sure thing, I already have a cool idea, will fix it in a moment.

wouterz commented:
LGTM

dmalarev commented:
Rest looks good

Commit:
Remove Error Code

Removed the error codes of -2 and -3 to prepare for incoming changes to the validate system, as the purpose of them will not be in the final product.

Commit:
Refactor Validate

Refactored the validate method to lower the network traffic as per Jesse and Dmitry's advice.

dmalarev commented:
Looks good to me
